dashboard/src/components/Table/BootsTable.tsx

```diff
@@ -124,24 +110,35 @@ const BootsTable = ({ treeId, testHistory }: ITestsTable): JSX.Element => {
     }
 
     return filteredData?.slice(startIndex, endIndex).map(test => (
-      <TableRow onClick={() => onClickName(test.id)} key={test.id}>
-        <TableCell>{test.path}</TableCell>
-        <TableCell>{test.status}</TableCell>
-        <TableCell>
+      <TableRowWithLink
+        key={test.id}
+        linkProps={{
+          to: '/tree/$treeId/test/$testId',
+          className: 'grid grid-cols-[25%_15%_15%_25%_15%]',
```

Comment:
you can remove this to a const and reuse it in here and in the other tableRowWithLink, it will make it easier to change in the future